src/BloomExe/Browser.cs

```diff
@@ -196,7 +196,7 @@ public void SetEditingCommands( CutCommand cutCommand, CopyCommand copyCommand,
 			{
 				// Note: this is only used for the Undo button in the toolbar;
 				// ctrl-z is handled in JavaScript directly.
-				var result = RunJavaScript("calledByCSharp ? 'y' : 'f'");
+				var result = RunJavaScript("(typeof calledByCSharp === 'undefined') ? 'f' : 'y'");
```

Comment:
The original code, `calledByCSharp ? 'y' : 'f'`, throws a javascript error if calledByCSharp has not been initialized, which causes the return value to be null rather than 'f' and writes an error message to the console.